ogticrd/covid-safe-paths
# Feat: Using the public doc FAQs to display the data

#### Description:


Changing the source of the FAQ, instead of using a json to show the data I used the original doc in that way when something change in the document, it will update in the app too.
#### Linked issues:


Fix: [161](https://trello.com/c/Okd8AhvH/161-create-a-public-docs-for-faqs)
#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/86155875-0ef4a000-bad3-11ea-82f0-df6e56fe1975.png)
#### How to test:


- `git pull && git checkout Feat/161-create-public-doc-FAQs`
- Head over to settings and click on **Preguntas Frecuentes**
- Check how the FAQs are showing
